proxy/proxy/client.go

```diff
@@ -78,9 +81,12 @@ type Client struct {
 
 	// The cfgCache holds the most recent connection configuration keyed by
 	// instance. Relevant functions are refreshCfg and cachedCfg. It is
-	// protected by cfgL.
+	// protected by cacheL.
 	cfgCache map[string]cacheEntry
-	cfgL     sync.RWMutex
+	cacheL   sync.RWMutex
+
+	// cfgL prevents multiple goroutines from contacting the Cloud SQL API at once.
+	cfgL sync.Mutex
```

Comment:
Can we call this "refreshL" since it guards refreshCfg? (or refreshCfgL if you prefer)

Reply:
Done.